content/swiftui/concepts/views/terms/zstack/zstack.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'ZStack'
+Description: 'The ZStack view creates zed stacked elements, where elements lay in a back-to-front layout.'
```

Comment:
The first paragraph is clearer than this description. I'd use that verbiage instead,

Reply:
```suggestion
Description: 'The ZStack view is a layout container where elements, or subviews, are arranged as overlaying one another (back-to-front)'
```